svc-systems/README.md

```diff
@@ -1342,13 +1343,13 @@ curl -i GET \
 
 
 
-## Single chassis
+### Single chassis
 
 |||
 |---------|-------|
 |**Method** |`GET` |
 |**URI** |`/redfish/v1/Chassis/{ChassisId}` |
-|**Description** |This operation fetches information on a specific chassis.|
+|**Description** |This operation fetches information on a specific computer system chassis, rack group, or a rack.|
```

Comment:
make rackgroup a single word

Reply:
Checked in Redfish spec.  And this is the  description: A group of racks that form a single entity or share infrastructure.